src/collective/fingerpointing/upgrades/v3/__init__.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 from collective.fingerpointing.config import PROJECTNAME
-from collective.fingerpointing.logger import logger
+from collective.fingerpointing.logger import log_info
 
 
 def update_user_actions(setup_tool):
     """Update user actions."""
     profile = 'profile-{0}:default'.format(PROJECTNAME)
     setup_tool.runImportStepFromProfile(profile, 'actions')
-    logger.info('User actions updated.')
+    log_info.logger.info('User actions updated.')
```

Comment:
we don't want this information logged into the audit log, neither.

Reply:
see above.